trview.ui/TextArea.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include "StackPanel.h"
+
+namespace trview
+{
+    namespace ui
+    {
+        class Label;
+
+        /// A text area is a number of lines of text.
+        class TextArea final : public Window
```

Comment:
TextArea support could go in a separate commit as well - this would make the change for routes clearer.

Reply:
This was moved to issue #414.